### ISSUE TITLE
Default permalinks vs pretty permalinks

### ISSUE THREAD

**Issue by paaljoachim** [joedolson/my-calendar#49]
I am going through the guide.

I am testing pretty permalinks vs default permalinks.

Use Pretty Permalinks for Events
Adds the title of the event into the URL. 
http://yourdomain.com/mc-events/title-of-your-event/?mc_id=24

Turning off this option removes the title from the URL and only inserts the My Calendar slug.
http://yourdomain.com/my-calendar/?mc_id=6

What I find strange with the above is using **mc-events** with pretty permalinks and with default permalinks it uses **my-calendar**.

My assumption is that it would look like this:
Pretty permalinks:
http://yourdomain.com/my-calendar/title-of-your-event/?mc_id=24

Default permalinks:
http://yourdomain.com/my-calendar/?mc_id=6

With no mention of **mc-events**.

**mc events** should default to the My Calendar name or whatever name the user uses for the Calendar page.

**Comment by joedolson** [comment]
The difference here is whether the plugin uses the post page or not. Without pretty permalinks, the calendar renders all events through the main shortcode, which is on the page 'My Calendar', by default. That path would change if the page slug is changed.

This is largely historical, going back to before My Calendar used custom posts as a data store. 

Changing this will require two things: building a redirect from the shortcode renderer to the post permalink & adding an exclusion if the shortcode is calling the event directly, rather than calling by URL reference. 

This is made difficult mostly because the redirect has to happen before the shortcode is rendered.

**Comment by paaljoachim** [comment]
I seems we can just close the issue and call it a won't fix issue. As it seems like it is more hassle then what it is worth.

**Comment by joedolson** [comment]
It is a hassle, yes, and I'm with you - not really worth solving.
